src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -3,6 +3,18 @@ import styled, { css } from 'styled-components';
 import { edgeStyle, focusStyle, normalizeColor } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+const fillStyle = fillContainer => {
```

Comment:
```suggestion
const fillStyle = () => {
```
continuing from other comment around not needing any parameters here.

Reply:
Then below, just a single return with
```
return `
      width: 100%;
      height: 100%;
      max-width: none;
      flex: 1 0 auto;
`
```